pkg/server/event_handler_test.go

```diff
@@ -89,50 +90,58 @@ func TestEventHandler_HttpHandler(t *testing.T) {
 	ctx := context.Background()
 
 	cases := []struct {
-		name               string
-		pubsubMessageBytes []byte
-		gcsObjectBytes     []byte
-		wantStatusCode     int
-		wantRespBodySubstr string
+		name                   string
+		pubsubMessage          *PubSubMessage
```

Comment:
It looks confusing to me, why do you need both pubsubMessage and PubsubMessageBytes?

Reply:
pubsubMessageBytes is for test `invalid_request_body`, so we can directly return a [invalid request body](https://github.com/abcxyz/pmap/blob/add_pmap_event_info/pkg/server/event_handler_test.go#L124). 

pubSubMessage is for the other two tests, where we need an actual pubsubMessage.

Reply:
pubsubMessage.Message.Data has type `[]byte`. Initializing pubsub message would require we write code like 
```
PubSubMessageBytes:     []byte(`
{
"message": {
    "attributes": {
         "bucketId": "foo",
         "objectId": "bar2"
    },
    "data": [1 2 3 4 5 6 7 8]
}
`),
```
I think this is not good in terms of readability.

Reply:
Should we just have `pubSubMessageBytes`? It's the raw input we would get from the HTTP request anyway.

Reply:
Discussed with @sqin2019, use pubSubMessageBytes for all tests now.